app/src/main/java/org/wikipedia/search/RecentSearch.kt

```diff
@@ -0,0 +1,29 @@
+package org.wikipedia.search
+
+import java.util.*
+
+class RecentSearch @JvmOverloads constructor(val text: String?, val timestamp: Date = Date()) {
+
+    override fun equals(other: Any?): Boolean {
+        if (other !is RecentSearch) {
+            return false
+        }
+        return text == other.text
+    }
+
+    override fun hashCode(): Int {
+        return text.hashCode()
+    }
+
+    override fun toString(): String {
+        return ("RecentSearch{" +
+                "text=" + text +
+                ", timestamp=" + timestamp.time +
+                '}')
```

Comment:
Would it be possible to remove the `( )` from the `return`?

And also, the line `22` uses single quotes, would it be possible to change to double quotes?